src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="container">
+    <div class="product-cards" data-qa="card">
+      <div class="product-cards__items">
+        <div class="image product-cards__image">
+          <img
+          src="images/imac.jpeg"
+          alt="Apple A1419"
+          class="image__item"
+        >
+        </div>
+        <h1 class="product-cards__text">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h1>
+        <p class="product-cards__code">Код товара: 195434</p>
+        <div class="stars product-cards__rate">
+          <span class="stars__image"></span>
+          <span class="stars__image"></span>
+          <span class="stars__image"></span>
+          <span class="stars__image"></span>
+          <span class="stars__image stars__image--disactive"></span>
+          <div class="stars_text">Отзывов: 5</div>
+        </div>
```

Comment:
You need to reuse stars block from previous task here.
Additionally, you need to apply `stars--4` modificator and appropriate styles, so `stars--3`, `stars--2` would also potentially work correctly. Don't hardcode 4 active stars.